test/interpreter/function-vlookup.spec.ts

```diff
@@ -6,15 +6,15 @@ import {Sheet} from '../../src/Sheet'
 
 const sharedExamples = (builder: (sheet: Sheet, config?: Partial<ConfigParams>) => HyperFormula) => {
   describe('VLOOKUP - args validation', () => {
-    it('not enough parameters', () => {
+    it('not enough metadata', () => {
       const engine = builder([
         ['=VLOOKUP(1, A2:B3)'],
       ])
 
       expect(engine.getCellValue(adr('A1'))).toEqual(detailedError(ErrorType.NA))
     })
 
-    it('too many parameters', () => {
+    it('too many metadata', () => {
```

Comment:
```suggestion
    it('too many metadata', () => {
```
it('too many parameters', () => {